actions/tax-calculation/index.js

```diff
@@ -0,0 +1,130 @@
+/*
+Copyright 2024 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+
+const { Core } = require('@adobe/aio-sdk');
+const { HTTP_OK } = require('../../lib/http');
+const { webhookErrorResponse, webhookVerify } = require('../../lib/adobe-commerce');
+
+
+const TAX_RATES = Object.freeze({
+  EXCLUDING_TAX: [
+    { code: 'state_tax', rate: 4.5, title: 'State Tax' },
+    { code: 'county_tax', rate: 3.6, title: 'County Tax' },
+  ],
+  INCLUDING_TAX: [{ code: 'vat', rate: 8.4, title: 'VAT' }],
+});
+
+/**
+ *
+ * @param {object} params - method params includes environment and request data
+ * @returns {object} - response with success status and result
+ */
+async function main(params) {
+  const logger = Core.Logger('webhook-collect-taxes', { level: params.LOG_LEVEL || 'info' });
+  logger.info('Starting webhook');
+
+  try {
+    const { success, error } = webhookVerify(params);
+    if (!success) {
+      return webhookErrorResponse(`Failed to verify the webhook signature: ${error}`);
+    }
+    const operations = [];
+
+    params.oopQuote.items.forEach((item, index) => {
+      operations.push(...calculateTaxOperations(item, index));
+    });
+
+    logger.info(`Successful request: ${HTTP_OK}`);
+
+    return {
+      statusCode: HTTP_OK,
+      body: JSON.stringify(operations),
+    };
+  } catch (error) {
+    logger.error(error);
+    return webhookErrorResponse(`Server error: ${error.message}`);
+  }
+}
+
+function calculateTaxOperations(item, index) {
+  const taxesToApply = item.is_tax_included ? TAX_RATES.INCLUDING_TAX : TAX_RATES.EXCLUDING_TAX;
```

Comment:
Why if this taxes comes from another method like obtainTaxes(item) then, inside this method you can directly ```return item.is_tax_included ? TAX_RATES.INCLUDING_TAX : TAX_RATES.EXCLUDING_TAX```

But doing this you can left a comment inside the method like:

//Here will be placed the call to external tax service to obtain the rates. 

By this way a system integrator could easily use this template and know exactly what part they should customize.